ddtrace/tracer/tracer.go

```diff
@@ -369,5 +375,8 @@ func (t *tracer) sample(span *span) {
 	if rs, ok := sampler.(RateSampler); ok && rs.Rate() < 1 {
 		span.setMetric(sampleRateMetricKey, rs.Rate())
 	}
+	if t.rulesSampling.apply(span) {
```

Comment:
This logic reveals that apply is not the best name here. Can we use “keep” or “sample”?

Reply:
At the moment, the return value indicates whether rules were matched and applied to the span, not whether it was kept or sampled.
The fallback for now is to use priority sampler. In the future that may disappear - it'd apply the value of `sampleRate()` to things not matching rules.

Reply:
Oh, I see. Sorry, I thought we were in another function here.